examples/QmlBarcodeGenerator/qml/Theme.qml

```diff
@@ -0,0 +1,13 @@
+import QtQuick 2.12
```

Comment:
Theme can be defined as Singleton - there is no point in having more than one instance of such item